packages/sync-actions/src/customers.js

```diff
@@ -9,38 +14,50 @@ import * as diffpatcher from './utils/diffpatcher'
 
 export const actionGroups = ['base', 'references', 'addresses', 'custom']
 
-function createCustomerMapActions(mapActionGroup) {
-  return function doMapActions(diff, newObj, oldObj /* , options */) {
+function createCustomerMapActions(
+  mapActionGroup: Function,
+  config: SyncActionConfig
+): (diff: Object, newObj: Object, oldObj: Object) => Array<UpdateAction> {
+  return function doMapActions(
+    diff: Object,
+    newObj: Object,
+    oldObj: Object /* , options */
+  ): Array<UpdateAction> {
```

Comment:
I did this re-factoring, because my editor was screaming with warnings. I figured why not just do it as we go